tests/unit_tests/test_wave_06.py

```diff
@@ -76,16 +76,19 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
-    # Assertions should check:
-    # - That the results is truthy
-    # - That tai and jesse's inventories are the correct length
-    # - That all the correct items are in tai and jesse's inventories, including the items which were swapped from one vendor to the other
+    assert result 
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert item_c not in tai.inventory
+    assert item_a in tai.inventory
+    assert item_b in tai.inventory
+    assert item_f in tai.inventory
+    assert item_f not in jesse.inventory
+    assert item_c in jesse.inventory
+    assert item_d in jesse.inventory
+    assert item_e in jesse.inventory
```

Comment:
ou can combine lines 80 and 81 if you want:
assert len(tai.inventory) == 3 and len(jesse.inventory) == 3

You can use list destructing too - a pythonic way of doing it:

assert [item_a, item_b, item_f] == tai.inventory
assert [item_c, item_d, item_e] == jesse.inventory

https://medium.com/@umaramanat66/destructuring-list-in-python-like-javascript-f7d4c0968538